src/frontend/app.ts

```diff
@@ -279,17 +279,34 @@ export default (octokitApp: App): Express => {
 			next(err);
 		});
 	});
+
+	// Error endpoints to test out different error pages
+	app.get(["/error", "/error/:message", "/error/:message/:name"], (req: Request, res: Response, next: NextFunction) => {
+		res.locals.showError = true;
+		const error = new Error(req.params.message);
+		if(req.params.name) {
+			error.name = req.params.name
+		}
+		next(error);
+	});
+
 	// The error handler must come after controllers and before other error middleware
 	app.use(Sentry.Handlers.errorHandler());
 
 	// Error catcher - Batter up!
 	app.use(async (err: Error, req: Request, res: Response, next: NextFunction) => {
 		req.log.error({ payload: req.body, err, req, res }, "Error in frontend app.");
 
-		if (!isNodeProd()) {
+		if (!isNodeProd() && !res.locals.showError) {
 			return next(err);
 		}
 
+		// Check for IP Allowlist issue from Github to show the correct message
```

Comment:
I'm not a huge fan of comments explaining what code does, not why it does it. Could we maybe have a helper method "isIpAllowListError()" and use it instead of having this comment?

Reply:
Seems like overkill for a single condition only used here.  I can update the comment though

Reply:
Up to you, I'll just leave this link here ;) https://stackoverflow.blog/2021/07/05/best-practices-for-writing-code-comments/